relay-dynamic-config/src/defaults.rs

```diff
@@ -216,7 +225,7 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                     .always(),
                 Tag::with_key("span.domain")
                     .from_field("span.sentry_tags.domain")
-                    .when(is_db.clone() | is_resource.clone()),
+                    .when(is_db.clone() | is_resource.clone() | is_http.clone()),
```

Comment:
@gggritso Do we already know what tag combinations / filters we want to query `exclusive_time` by? Does this config accurately reflect those access patterns?

Reply:
We have initial product requirements, so we do know, even though it might change:

- `environment` for environment filters
- `span.domain`, to break down and filter by domain in the table, and possibly render top 5 domains charts
- `span.status_code`, to show 2xx, 4xx, and 5xx counts by span domain and/or by span transaction
- `transaction`, to show a table of transactions for a given span domain, filter by transaction
- `transaction.method` so we can filter by correct endpoint method in the sample panel

The config looks good to me 👍🏻